clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentWgsUbam.scala

```diff
@@ -38,8 +40,8 @@ case class DocumentWgsUbam(upsertId: UpsertId,
                            sampleType: Option[String] = None,
                            targetIntervals: Option[String] = None,
                            notes: Option[String] = None,
-                           ubamMd5: Option[String] = None,
-                           ubamPath: Option[String] = None,
+                           ubamMd5: Option[Symbol] = None,
+                           ubamPath: Option[URI] = None,
```

Comment:
Because of the generalization work we've done, it would have been a royal pain to change the gvcf and cram path types to `URI` but not change the type for ubams. This change should be safe because `URI` also maps to a keyword field, so the Elasticsearch mapping definition for this document won't change.

I also changed the md5 field to `Symbol` to match md5 fields in the other documents. It's compatible with the old index, too.